### ISSUE TITLE
Action bar: Change the text to "Unfollow" when a user follows a site/blog

### ISSUE THREAD

**Issue by filipanoscampos** [Automattic/wp-calypso#70232]
### What

When we follow a WordPress.com website/blog, by using the action bar, the text changes from "Follow" to "Following".
A user recently shared this was very confusing to them, and suggested we changed it to "Unfollow" instead, to make this clearer.

For clarification, here is what the user was referring to:
![followbutton](https://user-images.githubusercontent.com/2540430/203044549-e0d44ed1-83ca-4c31-a092-ded3981e8cb1.png)

More context, here: pd4QKa-Ed-p2

### Why

The text "Follow" and "Following" are very similar visually.
It also makes sense due to context.
[There is some great work being done to improve the action bar](https://github.com/Automattic/jetpack/issues/26394), this looks like the right time to act on this, given that it looks like the text will be as it is now:
![Action bar](https://user-images.githubusercontent.com/5634774/192012424-24f3db86-ec88-4beb-9669-4ea9bd06e222.png)

### How

By simply changing the text after a user follows a WordPress.com to show "Unfollow" instead of "Following" we would make it clearer to users that:

1. The button changed, and so did the action that clicking it causes
2. They can unfollow a followed blog at any time from there

**Comment by cuemarie** [comment]
📌 ACTIONS
- Feature request kept

**Comment by jromales** [comment]
7192652-zd-a8c - I had a user in chat who shared feedback: "The route to unsubscribe from another WordPress.com site takes way too much time and it is easier to just not subscribe."

Similar to this feature request, user would like to see the option to unsubscribe through the action bar when accessing that website they subscribed to, and not have to go through the steps. The current steps to unsubscribe are the following: 

1. Click on [Reader](https://wordpress.com/read) in the WordPress.com toolbar.
2. Select the Sites tab.
3. Click Manage.
4. Click the three dots next to the title of a specific blog you follow and choose Unsubscribe.

**Comment by davemart-in** [comment]
Moved to Loop team board.

**Comment by aaronrobertshaw** [comment]
I went to pick this issue up but it appears to have been addressed already as of this [old commit](https://github.a8c.com/Automattic/wpcom/commit/189b03fe91d3aeb6da425f767fbe8e0dd380f270) that migrated `Follow`/`Following` to `Subscribe`/`Subscribed`.

To me personally, it makes sense for a toggle button to indicate the state as it currently does. The action of toggling the subscription status also seems captured via the icon change seen below:

| Unsubscribed | Subscribed |
|---|---|
| <img width="173" alt="Image" src="https://github.com/user-attachments/assets/15d655c6-17dc-40b4-aa60-bff9476ddc6d" /> | <img width="181" alt="Image" src="https://github.com/user-attachments/assets/7d9f54ba-71ae-4c56-8531-973bd8235878" /> |


Given the current state of the subscribe button in the action bar, do we still wish to change this to `Subscribe`/`Unsubscribe`?

On this issue at least there doesn't appear to have been a lot of reports of this current state causing confusion. As it stands now, it also addresses the [one report above](https://github.com/Automattic/wp-calypso/issues/70232#issuecomment-1785729224). My vote would be to close this issue but I don't hold that opinion strongly.

cc/ @ramonjd do you have any thoughts given your recent work on the actionbar?

**Comment by ramonjd** [comment]
Thanks @aaronrobertshaw 

> My vote would be to close this issue but I don't hold that opinion strongly.

Yeah it seems to me to be low priority. Also from 2022, so I think it'd be okay to close. Folks can always reopen.

It'd have been good first issue though! Even so, this one reads like an actual bug, and seems straightforward, so might have higher priority: https://github.com/Automattic/wp-calypso/issues/86454

**Comment by aaronrobertshaw** [comment]
Yep, I'll close this. My thinking was that either way it should be quick to clean up.
